package.json

```diff
@@ -16,17 +16,16 @@
     "url": "git://github.com/venables/koa-helmet.git"
   },
   "engines": {
-    "node": ">= 0.11.16",
-    "iojs": ">= 1.0.0"
+    "node": ">= 4.0.0"
   },
   "dependencies": {
-    "helmet": "^0.10.0"
+    "helmet": "0.12.0"
```

Comment:
We should always use the caret `^` when mapping to a helmet version, so when they make changes minor, it does not require bumping koa-helmet.